sql/wsrep_sst.cc

```diff
@@ -1578,6 +1618,34 @@ int wsrep_sst_donate(const std::string &msg, const wsrep::gtid &current_gtid,
     return WSREP_CB_FAILURE;
   }
 
+
+  /*
+  [start]
+  section to support clone user/pw
+  check for auth@addr separator
+  */
+  const char* addr= strrchr(data, '@');
+  wsp::string remote_auth;
+  if (addr) {
+    remote_auth.set(strndup(data, addr - data));
+    addr++;
+  }
+  else {
+    // no auth part
+    addr= data;
+  }
+
+  /* Set up auth info (from <user>:<password> strings) */
+  sst_auth auth;
+  if (remote_auth()) {
+    /* wsp::string is just a dynamically allocated char* underneath
+     * so we can safely do all that arithmetics */
+    const char* col= strchrnul(remote_auth(), ':');
```

Comment:
Here and everywhere, the space around `=` is inconsistent. Please run clang-format before merging.
```suggestion
    const char* col = strchrnul(remote_auth(), ':');
```

Reply:
clang formatting for the whole file fixed